src/main/java/reactor/ipc/netty/http/client/HttpClientWSOperations.java

```diff
@@ -141,6 +144,11 @@ public void onInboundNext(ChannelHandlerContext ctx, Object msg) {
 		}
 	}
 
+	private boolean continueProcessing(HttpResponse response) {
```

Comment:
@violetagg @smaldini is using parent's `HttpClientOperations#checkResponseCode(HttpResponse)` an option, or is there a major difference in how things should be processed?